### PR TITLE
[WIP] update field name : Event.Event is now Event.Parsed

### PR DESCRIPTION
**Pull request by buixor**

### CODE REVIEW

File: parsers/s02-enrich/crowdsecurity/whitelists.yaml [hunk 1]
```diff
@@ -3,11 +3,12 @@ description: "Whitelist events from private ipv4 addresses"
 whitelist:
   reason: "private ipv4 ranges"
 #  ip: 
-#    - 195.154.37.115
+#    - 1.2.3.4
   cidr:
     - "192.168.0.0/16"
     - "10.0.0.0/8"
     - "172.16.0.0/12"
+    - "127.0.0.1"
```
Comment (erenJag) [inline parsers/s02-enrich/crowdsecurity/whitelists.yaml:11]: it's missing the CIDR, does it work ?

File: scenarios/crowdsecurity/http-bf-wordpress_bf.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ name: crowdsecurity/http-bf-wordpress_bf
 description: "detect wordpress bruteforce"
```
Comment (erenJag) [inline scenarios/crowdsecurity/http-bf-wordpress_bf.yaml:3]: Is it possible to rename this scenario, there is "bf" twice in the name.

File: parsers/s02-enrich/crowdsecurity/whitelists.yaml (hunk 1 above)
Comment (buixor) [reply parsers/s02-enrich/crowdsecurity/whitelists.yaml]: fixed and moved to IP

File: scenarios/crowdsecurity/http-bf-wordpress_bf.yaml (hunk 2 above)
Comment (buixor) [reply scenarios/crowdsecurity/http-bf-wordpress_bf.yaml:3]: mh not really, I like to keep `<proto>-<attack-type>-<scenario name>` for now

**Pull request merged**